roboteam_ai/src/skills/Attack.cpp

```diff
@@ -8,53 +8,56 @@ namespace rtt {
 namespace ai {
 
 Attack::Attack(string name, bt::Blackboard::Ptr blackboard)
```

Comment:
Maybe split this into smaller sub skills so others could use this

Reply:
maybe for later, for now keep it in one skill.